src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -401,16 +401,19 @@ func (b *ByocAws) bucketName() string {
 
 func (b *ByocAws) environment(projectName string) map[string]string {
 	region := b.driver.Region // TODO: this should be the destination region, not the CD region; make customizable
+	defangStateUrl := fmt.Sprintf(`s3://%s?region=%s&awssdk=v2`, b.bucketName(), region)
```

Comment:
Should we print a warning if `PULUMI_BACKEND_URL` is set and `DEFANG_PULUMI_BACKEND` is not set.

Reply:
Hmm yeah and similarly for `pulumi-cloud` but no `PULUMI_ACCESS_TOKEN`?